client/components/shared/constants/technologies.js

```diff
@@ -0,0 +1,9 @@
+export default [
+  'JavaScript',
+  'Ruby',
+  'Java',
+  'Python',
+  'PHP',
+  'C++',
+  'Other'
+]
```

Comment:
regarding this constants files, I would rather do something similar to actionTypes. Export all one by one and then in the place we want to use, I would create there the array.